proof/proof/src/TLockPath.cxx

```diff
@@ -35,10 +41,17 @@ Int_t TLockPath::Lock(Bool_t shared)
 {
    const char *pname = GetName();
 
+#if defined(R__WIN32) && !defined(R__WINGCC)
+   if (gSystem->AccessPathName(pname))
+      fLockId = _open(pname, _O_CREAT | _O_RDWR, 0644);
+   else
+      fLockId = _open(pname, _O_RDWR);
+#else
    if (gSystem->AccessPathName(pname))
-      fLockId = open(pname, O_CREAT|O_RDWR, 0644);
+      fLockId = open(pname, O_CREAT | O_RDWR, 0644);
    else
       fLockId = open(pname, O_RDWR);
+#endif
```

Comment:
These two branches look similar enough, we could `#define _O_CREAT O_CREAT`, etc, to avoid having an ifdef here. What do you think?

Reply:
I agree. Will test and commit changes